text/0000-editions.md

```diff
@@ -0,0 +1,86 @@
+
+Editions give us an opportunity to bring our documentation and marketing up-to-date to reflect the improvements we’ve made since the previous edition. They are also the time to synchronize Ember and the wider ecosystem, including updating default settings, blueprints, addons, and more.
+
+Editions also help ensure that Ember users are getting a cohesive, polished experience. The right time to declare a new edition is when:
+
+
+- A significant, coherent set of new features and APIs have all landed in the stable channel.
+- Error messages and the developer ergonomics of those new features have been fully polished.
+- Tooling (the Ember Inspector, blueprints, codemods, etc.) has been updated to work with these new features.
+- API documentation, guides, tutorials, and example code has been updated to incorporate the new features.
+
+## Motivation
+
+Ember releases a new, stable version every six weeks. For existing users, this drumbeat of incremental improvement is easier to keep up with than splashy, big-bang releases.
+
+However, this steady stream of small improvements comes with some downsides.
+
+- For Ember users, it’s hard to keep track of evolving best practices, and understand how new features are supposed to work together.
+- For non-Ember users, it’s not obvious what has changed and whether Ember is worth another look.
+- For Ember contributors, it’s not clear when to update documentation and marketing, because there’s always more features in the pipeline.
+
+This RFC proposes a new release concept called “editions”, which provides a lens through which to view these accumulated changes. It also marks an opportunity to synchronize the wider ecosystem with the latest changes to Ember, including updates to defaults, documentation, addons, and more.
+
+## Detailed design
+
+Editions touch several parts of the Ember ecosystem, and generally tie together a set of incremental changes with a unifying theme. Most of these changes should come from existing community processes (i.e. RFCs), with editions serving as a higher level conceptual grouping.
+
+### Editions vs. Releases
+Editions are not separate releases, but rather a specific range of releases belong to an edition. The current edition will be open-ended - it’s terminating release is determined by whenever the next edition is declared.
+
+### Proposing Editions
+Each edition is proposed via a RFC, which should describe in detail the change in programming model and how to communicate that with users.
+
+### Features
+Editions mark an opportunity to focus on shipping a set of thematically related feature work. For example, the first edition, Ember Octane, focuses on “performance and productivity”. It includes a few larger features as well as several smaller ones, all focused on delivering that theme. Note that editions are backwards compatible as well - they are decoupled from major semver releases.
```

Comment:
Comparing your comments here to this earlier statement
> Editions are not separate releases, but rather a specific range of releases belong to an edition 

I'm a little confused as to whether there's a possibility of an edition that crosses a major release boundary
```
-------Ember Octane-----|-----Ember Butane---
--3.14----3.15--|--4.0---4.1---4.2----4.3----
```

Reply:
Yes, editions (as proposed) can cross major version boundaries. That's because, due to Ember's approach to handling semver, major versions do not necessarily indicate a change in "programming model" - only that deprecated functionality has been removed.